main/cli.c

```diff
@@ -1948,6 +1949,63 @@ static char *group_show_channels(struct ast_cli_entry *e, int cmd, struct ast_cl
 #undef FORMAT_STRING
 }
 
+static char *group_show_variables(struct ast_cli_entry *e, int cmd, struct ast_cli_args *a)
+{
+#define FORMAT_STRING     "%-20s  %-20s\n"
+#define FORMAT_STRING_VAR "     %s=%s\n"
+
+	struct ast_group_meta *gmi = NULL;
+	struct varshead *headp;
+	struct ast_var_t *variable = NULL;
+	int numgroups = 0;
+	char group[80];
+	char category[80];
+
+	switch (cmd) {
+	case CLI_INIT:
+		e->command = "group show variables";
+		e->usage =
+			"Usage: group show variables [group@[category]]\n"
+			"       Lists all currently active groups and their variables.\n"
+			"       Optional group, or group@category can be used to only show a specific group\n";
+		return NULL;
+	case CLI_GENERATE:
+		return NULL;
+	}
+
+	if (a->argc < 3 || a->argc > 4) {
+		return CLI_SHOWUSAGE;
+	}
+
+	if (a->argc == 4) {
+		if (ast_app_group_split_group(a->argv[3], group, sizeof(group), category, sizeof(category))) {
+			return NULL;
+		}
+	}
+
+	ast_cli(a->fd, "Group   Category\n   Variables\n");
```

Comment:
The title format does not line up width-wise with the contents. Make Category line up with the categories.
Variables makes sense if there are any variables but otherwise it might be confusing.

```
ess*CLI> group show variables
Group   Category
   Variables
PJSIP/gxw9            deviceinuse
1                     lineinuse
1                     oe
     master=PJSIP/gxw9-0000000c
1                     oeall
1                     cdpr
5 active groups
```

Reply:
Fixed